src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -459,4 +459,18 @@ describe('sidebar/store/modules/annotations', function () {
       });
     });
   });
+
+  describe('#annotationExists', () => {
```

Comment:
Have we bottomed out on a convention for using `#` (or not) to denote methods/selectors? I omit it at present, FWIW.

Reply:
I used `#` here because we're accessing the function as a method rather than a function. I don't feel too strongly either way ... maybe let's hash out a convention on Slack?